infrastructure/parameters/dev.parameters.json

```diff
@@ -20,6 +20,12 @@
     "publishReleaseContentCronSchedule": {
       "value": "0 30 * * * *"
     },
+    "azureDataFactoryName": {
+      "value": "s101d01datafactory"
+    },
+    "azureDataFactoryPipelineName": {
+      "value": "pl_release_statistics"
```

Comment:
This name probably won't change per environment so I'm going to remove this override in `dev.parameters.json`